blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/astyanax/AstyanaxIO.java

```diff
@@ -116,7 +116,7 @@ private static ConnectionPoolConfigurationImpl createPreferredConnectionPoolConf
         return connectionPoolConfiguration;
     }
 
-    protected static Keyspace getKeyspace() {
+    public static Keyspace getKeyspace() {
```

Comment:
I need it to set up a mutation batch for LocatorIOIntegrationTest to call astyanaxWriter.insertLocator.  That test class is in package com.rackspacecloud.blueflood.io, and test both Datastax and Astyanax stuff.  Do you rather we get rid of that test there?  (see test writeOldAstyanaxReadDatastax in LocatorIOIntegrationTest).  Or leave this method public?